### ISSUE TITLE
Add support for non-fixed (static) header

### ISSUE THREAD

**Issue by seanmadsen** [squidfunk/mkdocs-material#205]
I'm customizing Material with some additional CSS and one of my goals is to change the site header (blue bar at the top) to be non-fixed &mdash; that is, I'd like it to move up (and out of view) when I scroll the page. 

The following CSS gets me most of the way there: 

```css
header.md-header {
  position: static;
}

.md-main__inner {
  margin-top: 0;
}
```

*But* the perfectionist in me isn't satisfied with the extra vertical space that remains above the primary and secondary sidebars after scrolling. Is there an easy way I can get those sidebars to continue moving up just a bit more, after the header has been scrolled up? 

*In case you're curious: here's why I want to change the header positioning... I'm using Material for a site with several dozen books, and I'd like to add an additional header above the standard header which will allow users to navigate between the various books and see which book they're on. With this additional content in the header, I think it would be a hog of vertical space to keep it fixed. (But I will say: I do like the design choice to keep it fixed by default, it just doesn't suit my custom needs perfectly.)*

**Comment by squidfunk** [comment]
Good point. I've some refactoring in the pipeline that will make this very easy. Will keep this issue updated with notes on this topic.

**Comment by squidfunk** [comment]
Fixed in master, will be released with 1.3.0. The following `extra_css` is necessary:

``` css
.md-container,
.md-main__inner {
  padding-top: 0;
}

.md-header {
  position: static;
}

.md-sidebar[data-md-state=lock] {
  top: 0;
}
```

**Comment by squidfunk** [comment]
1.3.0 was just released.
